### DISCUSSION TITLE
Confusion about Composables

### DISCUSSION THREAD

**Discussion by abitwhy** [vuejs/docs#1955] [Q&A]
![image](https://user-images.githubusercontent.com/68452982/189794810-ca37bb9b-b546-4a25-a390-65148b4da430.png)

What's the stateful logic of the `useEventListener` in [Mouse Tracker Example](https://vuejs.org/guide/reusability/composables.html#mouse-tracker-example)? It looks like a normal function.

**Comment by splincool** [comment]
In my opinion, Composables are usual normal functions. This example shows us how easy we can divide some code logic into small composables(functions) and reuse this logic into components or other composables.

**Comment by AloisSeckar** [comment]
I'd say it is because you are altering the `target` object by adding/removing event listener to it.

But TBH, so far I am struggling to divide composables/utils properly in my main work project. It is fairly easy to mix them up during development and then quite hard to split them again...
